src/zsales_cl_abapgit.clas.abap

```diff
@@ -12,6 +12,7 @@ CLASS zsales_cl_abapgit DEFINITION
         matnr TYPE vbap-matnr,
         charg TYPE vbap-charg,
         werks TYPE vbap-werks,
+        zmeng TYPE vbap-zmeng,        "SCR1 - 02.07.2020 - V1
```

Comment:
New Field ZMENG added - as per FD - Ok
Please add unit of measure field MEINS from VBAP - Review Comments

Reply:
Unit of Measure added - Please Review